client/my-sites/checkout/composite-checkout/composite-checkout.js

```diff
@@ -499,10 +500,9 @@ export default function CompositeCheckout( {
 	);
 
 	return (
-		<React.Fragment>
+		<CalypsoI18nProvider>
```

Comment:
This is fine for testing, but I don't think we should be putting this in the middle of the app. It should be part of each Calypso entrypoint.